migrace/072.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$this->q("UPDATE `stranky` SET `obsah` = 'Další vlna nových aktivit: <b>1.7. v 20:00</b>', `url_stranky` = 'program-legenda-text' WHERE `stranky`.`id_stranky` = 131;");
```

Comment:
Jelikož ID je autogenerované, tak ho neřídíme a svým způsobem by to na každé kopii Gameconu mohlo trefit jiný záznam.
Hodil bych sem podmínku podle původní URL stránky, která pochází od nás:
```suggestion
$this->q("UPDATE `stranky` SET `obsah` = 'Další vlna nových aktivit: <b>1.7. v 20:00</b>', `url_stranky` = 'program-legenda-text' WHERE `url_stranky` = 'program-legenda'");
```

Reply:
pak to je teda špatně ají v 032.php

Reply:
Ono to není špatně, jen to není ideálně.-) A já jsem hnidopich.